pull_request_template.md

```diff
@@ -1,14 +1,23 @@
 **References** <!-- Link to the Jira task -->
 PROJ-XXX
 
-### Changed <!-- What changed with this PR -->
+### Type of change
+- [ ] Bug fix <!-- non-breaking change which fixes an issue -->
+- [ ] New feature <!-- non-breaking change which adds functionality -->
+- [ ] Breaking change <!-- fix or feature that would cause existing functionality to not work as expected -->
 
-### Visuals <!-- Add here some screenshots -->
+### Description of the change with screenshots (if possible)
+<!-- Describe here what changed with this PR -->
 
-### How can this be validated? <!-- Instructions for QA -->
+### Validation steps
+<!-- Instructions for QA -->
 
 ### Pull request checklist
-- [ ] Can it be deployed automatically? _If manual actions are required:_ did you describe the deploy steps?
-- [ ] Is the documentation updated, if it makes sense? <!-- Check this also if no updating is needed -->
-- [ ] Are necessary translations added/updated? <!-- Check this even if no translations are being changed -->
-- [ ] Did you update sanitization routines, where personal information is handled?
+- [ ] Can it be deployed automatically? <!-- If manual actions are required, fill in the deploy steps below -->
```

Comment:
I'm not sure I agree with moving it to a comment like this. I don't know how other people use this, but personally I use the checklist to click the different checkboxes when it's in rendered mode. In this mode, the comments does not appear, and then I would not be reminded about filling it out. What do you think?